.github/workflows/crucible-go-build.yml

```diff
@@ -77,10 +92,16 @@ jobs:
           SOLVER_PKG_VERSION: "snapshot-20210917"
           BUILD_TARGET_OS: ${{ matrix.os }}
 
+      - name: Setup Environment Vars (GHC 8.6.5)
+        if: matrix.ghc == '8.6.5'
```

Comment:
All the `crucible-go` jobs work on CI now, which is exciting. I'm confused as to why this works, however. The `cachix/install-nix-actions` steps are divided up by whether the GHC version is 9.0.2 or not, but on the other hand, the `Setup Environment Vars` steps are divided up by whether the GHC version is 8.6.5 or not. Why can't they be the same?